pkg/codegen/schema/schema.go

```diff
@@ -610,7 +621,9 @@ type TypeSpec struct {
 	// Ref is a reference to a type in this or another document. For example, the built-in Archive, Asset, and Any
 	// types are referenced as "pulumi.json#/Archive", "pulumi.json#/Asset", and "pulumi.json#/Any", respectively.
 	// A type from this document is referenced as "#/types/pulumi:type:token".
+	// A type from another document is referenced as "/provider/vX.Y.Z/schema.json#/types/pulumi:type:token".
```

Comment:
Note that the path shouldn't need to be rooted.

Reply:
Can you clarify? I don't understand this comment.

Reply:
`/provider/vX.Y.Z/schema.json` is a rooted path, `provider/vX.Y.Z/schema.json` is not (basically, URLs allow the preceding slash to be emitted). Both forms should be allowed, especially b/c we use one for `pulumi.json` :)